ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -43,13 +43,32 @@ type ssh_gssapikexalgorithms = choice(
 );
 
 function is_valid_ssh_MAC = {
```

Comment:
Away from useful workstation so can't easily check, but can choice be made a list like other types?

```
type foo = {
   a : string[] # list of strings
   b : choice('x', 'y')[]  # list of choice types
};
```

Reply:
AFAICT yes that works, although maybe one for further cleanup and refactoring.